app/src/main/java/ch/epfl/sweng/erpa/activities/CreateGameActivity.java

```diff
@@ -74,7 +81,7 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        DataBindingUtil.setContentView(this, R.layout.activity_create_game);
+        setContentView(R.layout.activity_create_game);
```

Comment:
Pourquoi t'as changé ça ?

Reply:
i change that because of the menu. When we use DataBindingUtil it adds a "title" to the activity but what we want is a SupportActionBar (to add the menu button)